### ISSUE TITLE
Akademischer Titel in Publish-Formulare einbauen

### ISSUE THREAD

**Issue by alw-bsz** [OPUS4/application#1255]
Da immer wieder der akademische Titel bei Personen falsch im Vor- oder Nachname-Feld erfasst wird, soll das Feld "Akademischer Titel" im Default in den Publish-Formularen bei den Personen vorhanden sein.

**Comment by j3nsch** [comment]
Ja, im Augenblick ist das manuelle Hinzufügen zu allen Standard-Dokumenttypen die einzige machbare Lösung, aber hier zeigen sich auch wieder die Schwächen bzw. Lücken im Design. Das muss in der näheren Zukunft angegangen werden.

**Comment by alw-bsz** [comment]
Die Übersetzungen für die Feldnamen haben noch gefehlt.

**Comment by j3nsch** [comment]
Das ist erledigt.
